### ISSUE TITLE
Install on Windows 98 SE = Hang on boot?

### ISSUE THREAD

**Issue by kingdwemen** [itsmattkc/dotnet9x#5]
I tried installing this on my Windows 98 SE machine (probably very stupid) and now it hangs when it tries to boot. I'm able to get it to start in safe mode. How do I go about uninstalling? Would a whole OS re-install be in order?

**Comment by kingdwemen** [comment]
Wow, thanks for the quick reply! Today's video was great.

Just tried that, no luck. However, I just tried starting Windows with the "step-by-step" confirmation mode and the first thing I did was skip "processing the system registry." I said yes to everything else (load drivers and devices, run AUTOEXEC, etc.)and the machine booted! I was able to browse the internet and even play a couple games.

I had some issues with the registry on this machine before, so I wonder if that has something to do with it.
